newIDE/app/src/UI/ClosableTabs.js

```diff
@@ -173,9 +175,18 @@ export function ClosableTab({
   );
 
   const gdevelopTheme = React.useContext(GDevelopThemeContext);
-  const textColor = !active
-    ? gdevelopTheme.closableTabs.textColor
-    : gdevelopTheme.closableTabs.selectedTextColor;
+  const textColor = active
+    ? gdevelopTheme.closableTabs.selectedTextColor
+    : gdevelopTheme.closableTabs.textColor;
+
+  const lightness =
+    gdevelopTheme.palette.type === 'dark'
+      ? active
+        ? 0.978
+        : 0.776
+      : active
+      ? 0.022
+      : 0.224;
```

Comment:
That's highly specific values 😅  Could you add a comment to explain how they were calculated?

Reply:
I divided by 255. I don't think it needs a comment.

Reply:
Because brightness is usually between 0 and 1 and the colors was in hexa from 0 to FF.
But it doesn't really matter, if the theme is changed it wont break anything and nothing forbid to adjust it manually if they are not comfortable with color representation.